modules/microshift-install-bootc-build-image.adoc

```diff
@@ -103,8 +109,8 @@ $ sudo podman images "${IMAGE_NAME}"
 ----
 +
 .Example output
-[source,text]
+[source,text,subs="attributes+"]
 ----
 REPOSITORY                       TAG         IMAGE ID      CREATED        SIZE
-localhost/microshift-4.17-bootc  latest      193425283c00  2 minutes ago  2.31 GB
+localhost/microshift-{product-version}-bootc  latest      193425283c00  2 minutes ago  2.31 GB
```

Comment:
This is not rendered correctly.

Reply:
The "Branch build" render is what _should_ be correct once published. That's the OCP default for the previews, usually.